README.md

```diff
@@ -52,6 +64,16 @@ In addition to the required packages for the ROS2 gem, you will also need some a
     sudo apt install python3-colcon-common-extensions
     ```
 
+### Libraries
+
+The o3de_kraken_orchestration package uses the [python-statemachine](https://pypi.org/project/python-statemachine/) library.\
+You can install it by running the following command
+```bash
+pip install python-statemachine
+```
+
+**Note:** Make sure you have the pip Python package manager installed.
+
 ## Installation ##
 
 - Use the [roscon_2022](https://github.com/aws-lumberyard-dev/o3de/tree/roscon_2022) branch of the `O3DE`.
```

Comment:
I think that the address of this repo was updated recently.

Reply:
Fixed.